qarakusi.am.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import importlib
+import sys
+
+
+def main():
+    if len(sys.argv) != 2:
+        sys.exit('Usage: qarakusi.am.py <problem-number>')
+
+    problem_id = sys.argv[1]
```

Comment:
Can we use argparse instead?

https://docs.python.org/3/library/argparse.html

Reply:
We can, my plan was to replace command-line parsing with a proper command-line library through a separate ticket, in the spirit of keeping individual PR scope small. What do you think about that?

Reply:
Filed https://github.com/qarakusi-am/qarakusi.am/issues/6

Reply:
Makes sense